src/components/VitalStat.vue

```diff
@@ -0,0 +1,136 @@
+<script>
+/**
+ * Displays one stat with @param stat.id as a name of stat
+ * Then @param scale number of points with css style @param stat.style
+ * And init and fill css style depending on computed @param value while looking
+ * at @param stat.defaultValue and @param stat.defaultModifier
+ * @param stat {JSON} displayed object
+ * @param scale {Number} total number of points, split into groups of five
+ * @param value {Number} sum of dependency values
+ */
+export default {
+  props: { stat: Object, scale: Number, value: Number },
+  data() {
+    return {
+      // initalValue is stat.defaultValue or this.finalValue
+      // finalValue is stat.defaultModifier + value or value
+      initialValue: this.stat.defaultValue
+        ? this.stat.defaultValue
+        : this.finalValue()
+    };
+  },
+  methods: {
+    /**
+     * @returns computed modifier + value
+     */
+    finalValue() {
+      return this.stat.defaultModifier + this.value
+    },
+    /**
+     * @returns keyed array of class for a point with value @param valueOfPoint
+     */
+    classOfPoint(valueOfPoint) {
+      let classOfPoint = {};
+      classOfPoint[this.stat.style] = true;
+      classOfPoint['fill'] =
+        valueOfPoint > this.initialValue && valueOfPoint <= this.finalValue();
+      classOfPoint['init'] = valueOfPoint <= this.initialValue;
+      return classOfPoint;
+    },
+  },
+};
+</script>
+
+<template>
+  <div class="box">
+    <div class="name">{{ stat.id }}:</div>
+    <div class="points">
+      <span v-for="i in scale" :key="i" :class="classOfPoint(i)"> </span>
+    </div>
+  </div>
+</template>
+
+<style scoped>
+.point {
+  cursor: default;
+}
+.points > :nth-child(5n + 0) {
+  margin-right: 10px;
+}
+.healthPt,
+.willPt {
+  width: 16px;
+  height: 20px;
+  display: inline-block;
+  vertical-align: middle;
+  text-align: center;
+  margin: 1px;
+  border: 1px;
+  border-style: solid;
+  border-color: transparent;
+}
+.hungerPt,
+.humanityPt {
+  width: 14px;
+  height: 14px;
+  border: 1px;
+  border-style: outset;
+  display: inline-block;
+  vertical-align: middle;
+  text-align: center;
+  margin: 1px;
+}
+.healthPt.fill {
+  background-color: #ff6666;
+}
+.healthPt.init {
+  background-color: #cc0000;
+  color: #999999;
+  border: 1px;
+  border-style: outset;
+}
+.willPt.fill {
+  background-color: #6666ff;
+}
+.willPt.init {
+  background-color: blue;
+  color: #999999;
+  border: 1px;
+  border-style: outset;
+}
+.humanityPt {
+  color: #666666;
+}
+.humanityPt.fill {
+  background-color: blue;
+}
+.humanityPt.init {
+  background-color: black;
+  color: white;
+}
+.humanityPt:hover:after,
+.healthPt.init:hover:after,
+.willPt.init:hover:after {
+  content: '/';
```

Comment:
toto sa nezobrazi pre vypocitanu cast - ruzovy bod zdravia sa nepreskrtne

Reply:
ano to je v poriadku, z hladiska ked si v aktivnej hre pridavas bod, tak by si ho nemal byt schopny ovplivnit dokial si ho nepridas, neviem, prislo mi to logicke, ze dokial si to nepridas, nemal by si na to byt schopny sahat, lebo by to mohlo viezt k under(over)flow problemom